sqlglot/dialects/hive.py

```diff
@@ -480,6 +480,13 @@ class Generator(generator.Generator):
             exp.VolatileProperty: exp.Properties.Location.UNSUPPORTED,
         }
 
+        def create_sql(self, expression: exp.Create) -> str:
+            # remove Unique Column Constraints
+            for constraint in expression.find_all(exp.UniqueColumnConstraint):
+                if constraint.parent:
+                    constraint.parent.pop()
+            return super().create_sql(expression)
```

Comment:
This looks iffy, we already have an entry for `exp.Create` in `TRANSFORMS`.. I'd rather remove that entry and have this method return `create_with_partitions_sql(...)` instead.

Reply:
Spark2 also overrides `create_sql` to return `create_with_partitions_sql`, should we instead fall back to Hive's `create_sql` method?

Reply:
Ok, makes sense to return `create_with_partitions_sql`

Reply:
My first attempt to return `create_with_partitions_sql(...)` results in an infinite loop. I'll look at this more closely later in the day, though at first glance looks like `generator:create_with_partitions_sql(...)` calls `create_sql`